libc/test/src/__support/math_extras_test.cpp

```diff
@@ -91,7 +91,7 @@ TYPED_TEST(LlvmLibcBitTest, FirstTrailingZero, UnsignedTypesNoBigInt) {
 }
 
 TYPED_TEST(LlvmLibcBitTest, FirstTrailingOne, UnsignedTypesNoBigInt) {
-  EXPECT_EQ(first_trailing_one<T>(cpp::numeric_limits<T>::max()), 0);
```

Comment:
Thanks for the patch! How about we keep the existing test cases (since they are correct, IIUC), but then add the additional test for zero? Here and below.

Reply:
They are not correct. The max value should make the function return 1 (since the least significant bit is set), not 0, but I can add them both.

Reply:
Please do!